pkgs/tools/misc/handlr/default.nix

```diff
@@ -13,13 +13,19 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "sha256-xDQV8wVlzItz0lzR1nVRPVsg7nSf/khUhevDlGgSO3g=";
 
-  nativeBuildInputs = [ shared-mime-info ];
+  nativeBuildInputs = [ installShellFiles shared-mime-info ];
   buildInputs = lib.optional stdenv.isDarwin libiconv;
 
   preCheck = ''
     export HOME=$TEMPDIR
   '';
 
+  postInstall = ''
+    installShellCompletion \
+      --zsh  completions/_handlr \
```

Comment:
No bash?

Reply:
It appears nobody has contributed a bash completion script upstream yet. https://github.com/chmln/handlr/tree/master/completions